QuickDeployment/V2/QuickDeploymentDesktopReadme.md

```diff
@@ -0,0 +1,360 @@
+# Chocolatey Quick Deployment Environment README
+
+> :memo: **NOTE**
+>
+> This document is for **Version 2.0** of the Quick Deployment Environment.
+> If you're using an older version of QDE, please refer to the [[document for that version||QuickDeploymentEnvironment_v1]].
```

Comment:
Add see also common errors and resolutions for QDE v1.

Reply:
That link would be the same page with just an anchor tag anyway; if they're after QDEv1 stuff at all, they'll see that there.